src/index.js

```diff
@@ -41,10 +41,11 @@ app.use(methodOverride('_method'))
 app.use(expressValidator())
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: false }))
-app.set('port', process.env.PORT)
 app.set('view engine', 'html')
```

Comment:
need to read up on these other config lines, but we're definitely using the port one.